### ISSUE TITLE
Not showing Icon

### ISSUE THREAD

**Issue by koddek** [mrRundgren/BlazorMaterialSymbols#1]
Trying to use this library.
I put this in my MainLayout.razor file: 
`<IconManager Type="@IconType.Rounded"/>`

Then this in my index page: 
`<Icon Name="@Icons.CheckSmall"></Icon>`
`<Icon Name="@Icons.CheckCircle"></Icon>`

When I run the app, the icon does not show, instead it shows the name of both icons. "check_small" and "check_circle" respectively. Is there something I am missing? The library looks great and would love it if it works.

**Comment by mrRundgren** [comment]
Hey Koddek, 

This sounds strange, since i have been using this in several projects without issues. Lets see if we can solve it for you. First some questions:

1. Which version of the package are you using?
2. What type of blazor project are you running? Webassembly or Hosted?

Not that it should matter, but could you try the closed tag syntax like this:

<Icon Name="@Icons.Home" />

Best regards,

/Anders

**Comment by mrRundgren** [comment]
Tried to replicate this, by creating both blazor server app and blazor webassembly app, and sadly cannot reproduce the issue. the steps i took were:

1. create a new project
2. add a reference to the nuget package: BlazorMaterialSymbols
3. Added <IconManager <IconManager Type="@IconType.Rounded" /> to MainLayout
4. Added some icons <Icon Name="@Icons.CheckSmall" />, <Icon Name="@Icons.CheckCircle" />

![image](https://user-images.githubusercontent.com/47566445/236672298-260df61f-01ba-4e25-81a1-ac5a1d74ded3.png)

The result is as expected:
![image](https://user-images.githubusercontent.com/47566445/236672324-ab6e767c-28ed-4295-b8c9-914504171f4b.png)

As a bonus I also tried it in a Maui Hybrid app, with positive results.
